docs/How-To-Organize-AGM.md

```diff
@@ -0,0 +1,162 @@
+If a situation occurs that prevents that, it is acceptable to execute it virtually, as was done in 2021 and 2022 during the COVID-19 pandemic.
+
+<!-- TOC start -->
+
+* [Roles](#roles)
+* [Logistics Timeline](#logistics-timeline)
+  * [Three months prior](#three-months-prior)
+  * [Two months prior](#two-months-prior)
+  * [Four weeks prior](#four-weeks-prior)
+  * [Three weeks prior](#three-weeks-prior)
+  * [Two weeks prior](#two-weeks-prior)
+  * [10 days prior](#10-days-prior)
+  * [One week prior](#one-week-prior)
+  * [Day before](#day-before)
+  * [Day-of](#day-of)
+* [Pre-planning](#pre-planning)
+  * [Finding a Meeting Venue](#finding-a-meeting-venue)
+  * [Who Qualifies For AGM Travel Assistance](#who-qualifies-for-agm-travel-assistance)
+* [Ideas for future AGMs](#ideas-for-future-agms)
+  * [Meeting enhancements](#meeting-enhancements)
+  * [Day-of enhancements](#day-of-enhancements)
+
+<!-- TOC end -->
+
+## Roles
+
+Expected participants:
+
+|Who|Role|
+|---|---|
+|Project Leadership Committee (PLC)|Should be physically present if possible, dialed-in if not. Several members must be present in person to run the event. Several members, regardless, needed to provide content for meeting.|
+|Project Leader (PL)|Should be physically present if possible, dialed-in if not. Regardless, needed to provide content for meeting.|
+|Technology Steering Committee (TSC)|Should be physically present if possible, dialed-in if not. Regardless, needed to provide content for meeting.|
+|Members|Should dial-in or participate in person if possible.|
+
+PLC members' roles of responsibility for planning and execution:
+
+|Who|Role|
+|---|---|
+|Logistics Coordinator (LC)|Coordinates with meeting venue, restaurants, members, committees, vendors|
+|Agenda Coordinator (AC)|Coordinates agenda and content to be presented|
+|Technology Coordinator (TC)|Coordinates video conference audiovisual setup|
+
+:information_source: _(A person may have more than one role but one person should not have all roles.)_
+
+## Logistics Timeline
+
+Past practice and future intent is for AGM to coincide with [FOSDEM](https://fosdem.org "Free and Open Source Developers European Meeting"), which is held in Brussels, Belgium annually typically on the Saturday and Sunday of the fifth ISO-8601 week of the calendar year, calculable with:
+
+    ruby -rdate -e "s=ARGV[0].to_i;s.upto(s+4).map{|y|Date.commercial(y,5,6)}.each{|y|puts [y,y+1].join(' - ')}" 2024
+
+AGM should be held on the Friday before or the Monday following FOSDEM.
+
+:information_source: _Regenerate the dates for the WHEN lines in the next several headers
+using this quick command:_
+
+    ruby -rdate -e "YEAR=ARGV[0].to_i;puts ([[44,YEAR-1],[49,YEAR-1]]+(1.upto(4).map{|wk|[wk, YEAR]})).map{|wk,yr|Date.commercial(yr,wk).to_s}" 2024
+
+### Three months prior
+
+**When:** Week 44 of YEAR-1 :date: `2023-10-30`
+
+- [ ] LC: Seek venue through previous contacts or RFP.
```

Comment:
```suggestion
* [ ] LC: Seek venue through previous contacts or RFP.
```

does this work/is it valid Markdown? If not: I reckon we should just disable that particular markdownlint rule.

Reply:
TIL. I'll change it.